src/OpenOfficeConvert/OOStyles.cs

```diff
@@ -2384,7 +2384,7 @@ private void MasterPageCreation()
                     _writer.WriteAttributeString("style:name", "CopyRight_20_Page");
                     _writer.WriteAttributeString("style:display-name", "CopyRight Page");
                     _writer.WriteAttributeString("style:next-style-name", "TableofContents_20_Page");
-                    _writer.WriteAttributeString("style:page-layout-name", "pm7");
+                    _writer.WriteAttributeString("style:page-layout-name", "pm8");

```

Comment:
Let's add a comment to the line to make it clear what pm8 means (MasterPageOnLeft) (or we could use a constant which would be better since it would be in the code.

Reply:
Ok